### DISCUSSION TITLE
部署成功，来这里添加demo

### DISCUSSION THREAD

**Discussion by hiltay** [Zfour/hexo-circle-of-friends#165] [General]
部署成功的同学可以把地址留在这里哦~

**Comment by TNXG** [comment]
https://tnxg.loyunet.cn/friendcircle/

**Comment by Gahotx** [comment]
欢迎访问 |ू･ω･` )
https://gahotx.cn/fcircle/

**Comment by LanYunDev** [comment]
https://lanyundev.vercel.app/fcircle/
菜鸟的小站.

**Comment by zhheo** [comment]
https://blog.zhheo.com/moments/
使用服务器+sqlite的最简单方式成功部署～

**Comment by Colsrch** [comment]
https://colsrch.cn/fcircle/

**Comment by penndu** [comment]
https://dusays.com/fcircle/
Docker+MySQL的方式部署。

**Comment by Falling42** [comment]
https://falling42.top/fcircle/

**Comment by Nesxc** [comment]
https://www.nesxc.com/fcircle/
Server+MySQL
Wordpress部署

**Comment by kuole-o** [comment]
https://guole.fun/moments/
hexo+butterlfy，GitHub+vercel+leancloud

**Comment by zkeq** [comment]
https://icodeq.com/fcircle/
欢迎来玩（

**Comment by BeaCox** [comment]
<https://blog.beacox.space/fcircle/>
Hexo Volantis GitHub+Vercel+Leancloud最简单方式
欢迎来访以及互换友链!!!

**Comment by qxchuckle** [comment]
哎嘿https://www.chuckle.top/fcircle/

**Comment by xlenco** [comment]
欢迎访问 :
https://xlenco.eu.org/fcircle

**Comment by JayHrn** [comment]
欢迎访问
https://blog.jayhrn.com/fcircle/

**Comment by flow2000** [comment]
https://blog.panghai.top/fcircle/
github+vercel+mongodb

**Comment by autjuly** [comment]
https://kitel.top/

**Comment by ciraos** [comment]
https://blog.itciraos.cn/fcircle/

**Comment by 661111** [comment]
[https://661111.github.io/post/fcircle/](https://661111.github.io/post/fcircle/)

**Comment by Wangyuxuan08** [comment]
> 部署成功的同学可以把地址留在这里哦~

https://081113.xyz/fcircle/

**Comment by 612901** [comment]
> https://661111.github.io/post/fcircle/

使用github+MONGODB部署

**Comment by cajunlei** [comment]
[https://ll.sc.cn/fcircle/](https://ll.sc.cn/fcircle/)
使用github+sqlite+vercel成功部署~~

**Comment by moxv** [comment]
使用github+sqlite+vercel成功部署~~
欢迎来访！
https://seersu.me/friends/

**Comment by lizilong1993** [comment]
欢迎访问，互访互换友链😊
https://lizilong.netlify.app/friends/

**Comment by 92jling** [comment]
后端github+sqlite+vercel，前端typecho页面，成功部署
https://nie.su/links.html

**Comment by AdingApkgg** [comment]
欢迎造访！
https://blog.saop.cc/fcircle/

**Comment by AkimioJR** [comment]
欢迎访问~
https://blog.zuilang.tk/fcircle
搭建环境：GitHub+sqlite+vercel

**Comment by happyet** [comment]
https://lms.im/blogrolls
不亦乐乎

**Comment by insdram** [comment]
https://nuoea.com/links
使用极简模式部署。

**Comment by Gavin-Meskin-Chen** [comment]
vercel还是有点慢
https://blog.cansin.top/fcircle

**Comment by junpengzhou** [comment]
# 欢迎访问~:laughing:
> GitHub+Vercel+Mongodb模式部署友链朋友圈

效果传送门：[https://www.junpengzhou.top/fcircle/](https://www.junpengzhou.top/fcircle/)

## 效果截图

![image](https://github.com/Rock-Candy-Tea/hexo-circle-of-friends/assets/22519817/feb914c5-0db5-4606-ad7a-fad9086c9b4b)


# 我的部署记录文章
> [Github+Vercel+Mongodb部署友链朋友圈](https://www.junpengzhou.top/article/85fa82d5.html)

![image](https://github.com/Rock-Candy-Tea/hexo-circle-of-friends/assets/22519817/944c0cd4-2c1d-47ae-87b6-1e0450b7e331)

**Comment by ChrisonLeon** [comment]
https://blog.chrison.cn/fcircle.html

Typecho版，配置成功。docker+sqlite

**Comment by yuwen1278** [comment]
https://www.xiaoyaomeow.cn/fcircle/

**Comment by musnows** [comment]
https://blog.musnow.top/fcircle/
参考教程使用github+vercel+sqlite成功部署

**Comment by AGEN233** [comment]
欢迎访问 |ू･ω･` )
https://agen233.top/fcircle/
御枫林下的小站

**Comment by Elykia093** [comment]
欢迎访问~
https://blog.elykia.cn/fcircle/
参考教程使用github+vercel+sqlite成功部署

**Comment by OORTaka0825** [comment]
欢迎访问。
Github+vercel部署成功
https://oortaka.top/fcircle
内密心書
